php-checks/src/main/java/org/sonar/php/checks/TooManyMethodsInClassCheck.java

```diff
@@ -90,6 +92,39 @@ private int getNumberOfMethods(ClassTree tree) {
     return nbMethod;
   }
 
+  private static boolean classIsDBEntity(ClassTree tree) {
+    boolean foriegnMethodFound = Boolean.FALSE;
+
+    for (ClassMemberTree classMember : tree.members()) {
+
+      if (!classMember.toString().contains("get") && !classMember.toString().contains("set")) {
+
+        foriegnMethodFound = Boolean.TRUE;
+        break;
+      }
+    }
+
+    return foriegnMethodFound;
+
+  }
+
+  private static boolean classHasEntityAnnotation(ClassTree tree) {
```

Comment:
Please remove the empty lines in the methods. They are not needed for readability.